delta-lake/delta-spark350db143/src/main/scala/com/databricks/sql/transaction/tahoe/rapids/GpuLowShuffleMergeCommand.scala

```diff
@@ -1,1083 +0,0 @@
-/*
- * Copyright (c) 2024, NVIDIA CORPORATION.
- *
- * This file was derived from MergeIntoCommand.scala
- * in the Delta Lake project at https://github.com/delta-io/delta.
- *
- * Copyright (2021) The Delta Lake Project Authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.databricks.sql.transaction.tahoe.rapids
```

Comment:
So is the plan to add this back in later? Or is this really gone gone?

Reply:
From what I know, this was an experimental feature only added for 13.3 hence I removed it from Databricks 14.3. If there is a change and we decide to add it back, then we will add it. We will have to devise a way to make this work with the on-disk deletion vectors.